d3a_api_client/utils.py

```diff
@@ -1,15 +1,18 @@
-import os
-import traceback
 import ast
-import requests
 import json
 import logging
+import os
+import requests
+import traceback
 import uuid
 from functools import wraps
+
 from tabulate import tabulate
-from d3a_interface.utils import key_in_dict_and_not_none, get_area_name_uuid_mapping,RepeatingTimer
+
 from d3a_interface.constants_limits import JWT_TOKEN_EXPIRY_IN_SECS
 from d3a_api_client.constants import DEFAULT_DOMAIN_NAME, DEFAULT_WEBSOCKET_DOMAIN
+from d3a_interface.utils import get_area_name_uuid_mapping, key_in_dict_and_not_none, \
```

Comment:
Minor, but better to move the d3a_interface imports above the d3a_api_client imports. In general it is much more preferrable to have the local imports last.

Reply:
True, I wasn't considering `d3a_interface` as a 3rd party, but that's actually what it is wrt this repo 👍